README.md

```
@@ -6,6 +6,24 @@ NovaCrate is in active development; we welcome feedback, feature requests, and b
 
 ## [▶️ Open NovaCrate](https://kit-data-manager.github.io/NovaCrate/)
 
+<details>
+<summary>⚠️ There are two public instances</summary>
+
+Note that there are currently two public instances of NovaCrate:
+ - On GitHub Pages at [kit-data-manager.github.io/NovaCrate/](https://kit-data-manager.github.io/NovaCrate/)
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)